tools/testing/target_determination/heuristics/edited_by_pr.py

```diff
@@ -14,6 +15,18 @@
 from tools.testing.test_run import TestRun
 
 
+# Some files run tests in other test files, so we map them to each other here.
+# This is a map from file that runs the test to regex that matches the file that
+# contains the test. Test file with path test/a/b.py should of the form a/b.
+# Regexes should be based on repo root.
+ADDITIONAL_MAPPINGS = {
+    # Not actual test files but rather functions defined in run_test.py that run
```

Comment:
To clarify, they do become actual test files in the CI (they get generated and then deleted). Not sure if that changes anything

Reply:
Edited comment to be hopefully be more accurate